map/generate-offline-map-overrides/README.md

```diff
@@ -2,11 +2,11 @@
 
 Take a web map offline with additional options for each layer.
 
-![Image of generate offline map overrides](GenerateOfflineMapOverrides.png)
+![](GenerateOfflineMapOverrides.png)
```

Comment:
alt text needs to come back in (maybe this is already addressed with that other PR: same for other samples here)

Reply:
Yep, I've restored these here